NCG613a_Project1.ipynb

```diff
@@ -639,6 +639,646 @@
+       "      <td>463.570128</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4992</th>\n",
+       "      <td>E01032739</td>\n",
+       "      <td>945.0</td>\n",
+       "      <td>3762.0</td>\n",
+       "      <td>1044</td>\n",
+       "      <td>0.501916</td>\n",
+       "      <td>1621</td>\n",
+       "      <td>0.570019</td>\n",
+       "      <td>905.172414</td>\n",
+       "      <td>2320.789636</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4993</th>\n",
+       "      <td>E01032740</td>\n",
+       "      <td>945.0</td>\n",
+       "      <td>777.0</td>\n",
+       "      <td>1099</td>\n",
+       "      <td>0.445860</td>\n",
+       "      <td>1391</td>\n",
+       "      <td>0.634076</td>\n",
+       "      <td>859.872611</td>\n",
+       "      <td>558.590942</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "      LSOA Code  LSOA_Crime2011  LSOA_Crime2021  2011 Population  \\\n",
+       "4988  E01000001           945.0           179.0             1465   \n",
+       "4989  E01000002           945.0           244.0             1436   \n",
+       "4990  E01000003           945.0            81.0             1346   \n",
+       "4991  E01000005           945.0           509.0              985   \n",
+       "4992  E01032739           945.0          3762.0             1044   \n",
+       "4993  E01032740           945.0           777.0             1099   \n",
+       "\n",
+       "      2011 Non UK Percentage  2021 Population  2021 Non UK Percentage  \\\n",
+       "4988                0.326962             1478                0.408660   \n",
+       "4989                0.270195             1382                0.450796   \n",
+       "4990                0.324666             1613                0.429014   \n",
+       "4991                0.392893             1098                0.485428   \n",
+       "4992                0.501916             1621                0.570019   \n",
+       "4993                0.445860             1391                0.634076   \n",
+       "\n",
+       "      2011 Crime/1000 people  2021 Crime/1000 people  \n",
+       "4988              645.051195              121.109608  \n",
+       "4989              658.077994              176.555716  \n",
+       "4990              702.080238               50.216987  \n",
+       "4991              959.390863              463.570128  \n",
+       "4992              905.172414             2320.789636  \n",
+       "4993              859.872611              558.590942  "
+      ]
+     },
+     "execution_count": 103,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Now we merge the new data of City of London into the final crime data\n",
+    "\n",
+    "crime_final.update(crime_col)\n",
+    "crime_final[crime_final['LSOA Code'].isin(na_lsoa) == True]"
```

Comment:
Did you validate this data was included in the final crime_final dataset? Perhaps run head(5) or tail(5) in the next statement to ensure you see the LSOA's you inserted

Reply:
The last line actually shows exactly those 6 LSOAs that had no crime data before. I can add a tail(10) below this block just in case.

Reply:
![Snipaste_2024-03-24_02-13-46](https://github.com/markMaynooth/Data-Analytics-Project/assets/11791634/c6d84fde-3d8e-462a-9278-c25bfb021bf3)
This is how it shows on my laptop by the way.

Reply:
I'd suggest changing how we calculate the crime for the 6 LSOAs. Looks like they're very disproportionate per 2011 LSOA so I'd  suggest weighting them the same way as 2021 - see my comment in Teams